gatsby_fp/gatsby-node.js

```diff
@@ -38,13 +37,14 @@ exports.onCreateNode = async ({
       // err returns a string value with no ability to
       // get the status code without searching for it within the string
       (async function getStatusCodeUsingFetch() {
-        const response = await fetch(node.photo_url) 
+        console.log(err)
+        const response = await fetch(node.photo_url)
         if(response.status != 404 ){
           throw new Error(`Invalid Status Code [${response.status}] for photo_url of user ${node.pf_username}`)
```

Comment:
I know this should have been caught a long time ago, but we're finding and learning new things as we move forward with this project. I am not sure on what this error message is based on, it displays the same error message with any kind of error that happens in `createRemoteFileNode`, for example, I removed the `getCache` property from the object above, and it threw this error message. When I first saw the error message, I thought this might be related to overloading the People API, but since I have not changed `GATSBY_CONCURRENT_DOWNLOAD` this ensures that it has nothing to do with the People API overloading. After debugging and adding some console logs I found that the error was due to removing `getCache`. We should tweak the error handling here to better describe the error.